circle.yml

```diff
@@ -1,10 +1,15 @@
 dependencies:
+  pre:
+    # Force updating wget due to the current containers being too out of date
+    - sudo apt-get update
+    - sudo apt-get install wget
   override:
     - wget -O atom-amd64.deb https://atom.io/download/deb
     - sudo apt-get update
```

Comment:
This is duplicate I know, I've left it in here for when we can remove the `pre` section.